test/Azure.Functions.Cli.Tests/E2E/StartTests.cs

```diff
@@ -12,18 +11,25 @@
 using System.Net.Sockets;
 using System.Net;
 using System.Diagnostics;
-using System.Security.Cryptography;
 using Azure.Functions.Cli.Common;
-using Microsoft.Azure.WebJobs.Script.Description.DotNet.CSharp.Analyzers;
 
 namespace Azure.Functions.Cli.Tests.E2E
 {
-    public class StartTests : BaseE2ETest
+    public class StartTests : BaseE2ETest, IAsyncLifetime
     {
+        private int _port;
         private const string _serverNotReady = "Host was not ready after 10 seconds";
+
         public StartTests(ITestOutputHelper output) : base(output) { }
 
-        [Fact(Skip = "Flaky test")]
+        public async Task InitializeAsync()
+        {
+            // Set as default port for majority of tests
+            _port = 7071;
```

Comment:
Will using static ports throughout cause issues? What if something else is holding that port open?

Maybe we can instead search for a random available port?

Reply:
I was also wondering if this would be an issue - I can look into using a random open port, but I will need to update all the tests here that are referencing 7071 directly and it might be cleaner to open a new PR for that 🤔